accounting/off_chain.go

```diff
@@ -97,5 +99,18 @@ func offChainReport(invoices []*lnrpc.Invoice, circularPayments map[string]bool,
 		reports = append(reports, entry)
 	}
 
+	for _, payment := range payments {
```

Comment:
Also pre-existing but found during testing. In the loop above (in `OffChainReport`) we do `cfg.PaidSelf(payment.PaymentRequest)`. That fails for keysend payments where the payment request is empty.

Reply:
Nice catch, thanks for testing!